computing-resources/README.md

```diff
@@ -8,6 +8,7 @@ compute-canada
 data/README
 printer
 eduroam
+macos-email
```

Comment:
This section is about "Computing Resources", and to me this has nothing to do with computing resources, but rather to the configuration of personal laptop, hence i would put it [here](https://intranet.neuro.polymtl.ca/onboarding/computer-setup.html)

Same goes for the 'eduroam' section